pkg/console/models.go

```diff
@@ -0,0 +1,89 @@
+package console
```

Comment:
Would it be possible to avoid using a new layer of types and just use the console client types entirely?